IND/MarketIndices/BasketIndices.rdf

```diff
@@ -161,7 +162,7 @@
 			</owl:Restriction>
 		</owl:equivalentClass>
 		<skos:definition xml:lang="en">constituent of a basket of credit risks, such as an event represented in a credit index</skos:definition>
-		<skos:editorialNote xml:lang="en">OTPP Note:: There will be a specific reference entity and a specific instrument per index - and only one instrument per entity, per index. note that a credi index constituent may be in principal also be loans (as with CDS also), in which case some of the terms here would not apply (specifically, the reference security identity if the Reference Entity has not also issued any such), however it&apos;s not clear how this would be treated.</skos:editorialNote>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">There will be a specific reference entity and a specific instrument per index - and only one instrument per entity, per index.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
rdfs:label is inconsistent with the URI (extra "reference")

Reply:
fixed